src/include/sof/trace.h

```diff
@@ -112,6 +112,7 @@
 #define TRACE_CLASS_KPB		(28 << 24)
 #define TRACE_CLASS_SELECTOR	(29 << 24)
 #define TRACE_CLASS_SCHEDULE	(30 << 24)
+#define TRACE_CLASS_LL		(31 << 24)
```

Comment:
It looks like we have duplication here and in upai/user/trace.h, we should de-duplicate (in another PR).  Btw, best to call this SCHEDULE_LL

Reply:
@lgirdwood 
I've made #1170 for deduplication ```TRACE_CLASS_*``` defines. 
I will update this PR after merging #1170.

Reply:
@lgirdwood 
I've renamed to TRACLE_CLASS_SCHEDULE_LL. Can we merge it since we postpone #1170?